modules/service/src/main/resources/db/migration/V0880__calibration_reference_data.sql

```diff
@@ -0,0 +1,30 @@
+-- Add an enum to indicate if a target has a calibration role
+create type e_calibration_role as enum('twilight', 'photometric', 'spectrophotometric', 'telluric');
+comment on type e_calibration_role is 'The role of the target in a calibration.';
+
+alter table t_target
+drop column c_role cascade;
+
+alter table t_target
+add column c_calibration_role e_calibration_role null;
+
+-- Drop constraints/trigger that are no longer needed because c_role is dropped
+alter table t_asterism_target 
+drop constraint asterism_target_is_science;
+
+drop trigger t_target_role_immutable_trigger on t_target;
+
```

Comment:
Should we also get rid of the old `e_target_role`?

Reply:
Good observation, I'll remove it